featurebyte/query_graph/sql/aggregator/window.py

```diff
@@ -543,19 +555,21 @@ def merge_tiles_order_independent(
             Result of joining expanded request table with tile table
         inner_group_by_keys: list[Expression]
             Keys that the aggregation should use
-        merge_exprs: list[str]
-            Expressions that merge tile values to produce feature values
-        inner_agg_result_names: list[str]
-            Names of the aggregation results, should have the same length as merge_exprs
+        groupby_columns: Optional[list[GroupbyColumn]]
+            List of groupby columns
 
         Returns
         -------
         Select
         """
-        agg_exprs = [
-            alias_(merge_expr, inner_agg_result_name, quoted=True)
-            for merge_expr, inner_agg_result_name in zip(merge_exprs, inner_agg_result_names)
-        ]
+        groupby_keys = [GroupbyKey(expr=key, name=key.name) for key in inner_group_by_keys]
+        agg_exprs, _ = _split_agg_and_snowflake_vector_aggregation_columns(
```

Comment:
call the helper method in preparation for adding logic to support vector aggregations.